spec/media/audio.spec.js

```diff
@@ -0,0 +1,46 @@
+
+var nop = function(){}, empty = {};
+
+var proxyquire = require('proxyquire').noCallThru();
+var stubs = {
+    '../conf.js': empty,
+    '../log.js': empty,
+    '../utils.js': empty,
+    '../constants.js': empty,
+    'engine': {
+        isLocal: false,
+        isHttps: false,
+        createAudio: function() {
+            return {};
+        },
+        unsubscribeElementEvents: nop,
+        subscribeElementEvents: nop,
+        createSource: function() {
+            return {};
+        },
+        appendToBody: nop
+    },
+    '../resource_manager.js': empty
+};
```

Comment:
Very nice idea in general, but I suppose for other modules, rather than audio, engine mocks could appear quite huge :).On the other hand, it's probably a way faster (and cleaner?) to start testing this way, plus in the end, we'll get several mocks in every spec-file we could blend into one in the end. I like the fact every spec is independent this way.